src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -385,6 +448,11 @@ private void openVirtualSpout(
         // Generate our virtualSpoutId using the payload id.
         final VirtualSpoutIdentifier virtualSpoutId = generateSidelineVirtualSpoutId(id);
 
+        if (spout.hasVirtualSpout(virtualSpoutId)) {
+            logger.info("VirtualSpout {} is already running", virtualSpoutId);
```

Comment:
seems like this should at least be a warning?

Reply:
I thought this, but eventually this is going to be called at an interval and after the first pass this will always emit, so I think INFO might be better here.